src/main/java/com/algorand/algosdk/builder/transaction/AssetConfigureTransactionBuilder.java

```diff
@@ -39,7 +42,7 @@
     }
 
     @Override
-    protected Transaction buildInternal() {
+    protected void buildInternal() {
```

Comment:
I think this is OK because buildInternal is not public, but we should avoid adding the txn field. One way to do this would be by repurposing this method from "build" to "apply". Perhaps `protected void applyTo(Transaction txn)`.

Then build would be something like:
```
build() {
    Transaction txn = new Transaction();
    this.applyTo(txn);
}
```

No need to do that now though, it looks like the behavior here is correct, and I'm really happy to see those deprecated function calls go away.

Reply:
I like the idea. I think I just go ahead and make the change. 
Thanks.